admin_manual/release_notes.rst

```diff
@@ -10,6 +10,18 @@ static files
 <https://doc.owncloud.org/server/8.1/admin_manual/configuration_files/
 serving_static_files_configuration.html>`_ from your ownCloud server.
 
+LDAP issue: 8.2 uses the ``memberof`` attribute by default. If this is not 
+activated on your LDAP server you will see this message in your LDAP log: 
+``<= bdb_equality_candidates: (memberOf) not indexed``. Fix this by disabling 
```

Comment:
Better look for: user groups are not detected and "Error PHP Array to string conversion at /var/www/html/owncloud/lib/private/template/functions.php#36" appears.

The LDAP log messages states a missing index, not necessarily an issue.